### DISCUSSION TITLE
Github for email address<title>

### DISCUSSION THREAD

**Discussion by Edblue3** [community/community#46218] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

What GitHub tool do I need to use to find my email contacts and email addresses that were sent to my email from my forgotten email password that has no backup email and phone?

**Comment by jge162** [comment]
Not possible.

**Reply by Edblue3** [reply]
I mean what is the name of this online tool that can fetch this information

**Reply by jge162** [reply]
No idea

**Comment by Edblue3** [comment]
Do you know which website similar to github that can help me

**Reply by jge162** [reply]
Can’t help you. Maybe try GitHub support

**Comment by MostafaWaleed0** [comment]
Unfortunately, GitHub does not provide a tool to retrieve email contacts or addresses that were sent to your email. This information is not stored by GitHub and is not accessible through their platform. If you have lost access to your email account and don't have a backup email or phone, you may need to contact your email provider for assistance in recovering your account.

**Comment by Edblue3** [comment]
What if they can't help because I forgot my security question like the name
of my favorite childhood place?

**Reply by MostafaWaleed0** [reply]
1. Go to the GitHub password reset page at https://github.com/password_reset.
2. Enter the email address associated with your GitHub account.
3. Follow the instructions in the password reset email to create a new password.

If you are unable to reset your password using the email address associated with your account, you can contact GitHub Support for assistance. You may need to provide proof of identity to verify your account ownership.

It's also a good idea to update your security information in GitHub after resetting your password, to ensure that you'll be able to access your account in the future if you need to.

**Reply by jge162** [reply]
@mostafa-mw hes not asking about GitHub account unfortunately. He’s trying to recover an email from somewhere else.

**Comment by Edblue3** [comment]
Correct its from a Yahoo account
